app/public/custom_script.js

```diff
@@ -0,0 +1,29 @@
+const addScriptToHeader = (src) => {
+    const script = document.createElement('script');
+    script.src = src;
+    script.async = true; // Optional: Load the script asynchronously
+
+    document.head.appendChild(script);
+}
+
+addScriptToHeader("https://cdnjs.cloudflare.com/ajax/libs/uswds/3.8.1/js/uswds.min.js")
+
+MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
+
+let observer = new MutationObserver(function (mutations, observer) {
+    let accordionItems = document.getElementsByClassName("accordion_item")
+    for (let i = 0; i < accordionItems.length; i++) {
+        accordionItems[i].addEventListener("click", function () {
+            const data_id = this.dataset.id
+            const accordionButton = document.querySelector(`[aria-controls="${data_id}"]`)
+            accordionButton.setAttribute("aria-expanded", "true");
+            const accordionBody = document.getElementById(data_id)
+            accordionBody.removeAttribute("hidden");
+        });
+    }
+});
+
+observer.observe(document.body, {
```

Comment:
Is there a way to narrow this further to just the chat window where we expect the accordions to be generated?

Reply:
I've adjusted the element monitoring to root which is one level below the doc body. Unfortunately the DOM tree looks something like this:
<img width="592" alt="Screenshot 2024-12-16 at 5 03 57 PM" src="https://github.com/user-attachments/assets/530eb785-d707-4d58-ab73-25275430a62c" />
There's multiple divs under the `MuiStack-root` class where the new text should appear. We could use the `IntersectionObserver` api, but I don't see too much of a benefit to iterating over the classes and observing, over monitoring at the root level.